### DISCUSSION TITLE
buttons are not seen in Amiberry 5.6.

### DISCUSSION THREAD

**Discussion by 22marco77** [BlitterStudio/amiberry#1093] [Q&A]
Problem with Joy keys DragonRise only recognizes the axis but does not accept key input (a,b and etc..) for both DragonRise controllers, this happens only with AMIBERRY emulator version 5.6, before last update everything it worked fine. The other emulators and controllers work fine. Amiberry 5.6 on Retropie 4.8 (buster) latest firmware on RASPBERRY PI3 MOD B+. I put the DragonRise mapping in the gamecontrollerdb_user.txt file using jstes and with its id, even remapping the file from Amiberry using the REMAP key, everything stays the same, like it doesn't accept the new file assignment. This doesn't allow me to play or select the various options in the GUI, I'm forced to use only the keyboard. Please if anyone has any advice to give me it will be well accepted, I don't know how to proceed to solve the problem anymore.

**Comment by themightyninja** [comment]
Hello, I think I'm hitting the same issue. I'm using DragonRise controllers and whilst some of the button mappings are coming through (such as input_exit_emulator_btn & input_menu_toggle_btn), the GUI function input_reset_btn mapping isn't working. I can see it being set correctly in amiberry.log.

**Comment by giantclambake** [comment]
All things considered and after reading others' issues in these regards, I would suggest upgrading to amiberry v5.6.8 and retesting.
